spec.md

```diff
@@ -1374,7 +1374,7 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. |
+| credentials | object | A free-form hash of credentials that can be used by applications or users to access the Service Instance. This field MAY be omitted if the Service Broker either cannot return the credentials or does not want to for security reasons. Note that during an asynchronous bind operation the Service Broker MUST include this field in at least the first response after the Service Binding has been successfully created, otherwise the Platform will not have the credentials to pass along to the application/user. |
```

Comment:
@n3wscott does this address the async bind case?

Reply:
It's a little weird to have async bind information in this PR since it may be submitted before #334

Reply:
@mattmcneeney does that mean you want it back out and a reminder in #334 ?

Reply:
This addition introduces an inconstancy. If we allow the omission of the credentials here, we also have to allow the omission for the same reasons when a binding is (re-)created with the identical parameters (status code 200 OK).

Reply:
@MHBauer please make similar edits to the create per Florian's suggestion - then we'll discuss at the f2f

Reply:
I disagree. The door shuts eventually, but that doesn't mean it's shut from the start.

For synchronous binding the credentials can be returned the once before becoming irretrievable.

For async binding we have to preserve the response for only so long.

Reply:
I feel like this is very much in conflict with stateless-ness. It seems like it would now be saying "must be preserved forever". Being able to do this now implies that the credential generation is deterministic. I'm not sure this is clarifying anything anymore.

Reply:
What's the equivalent of ` MUST include this field in at least the first response`? We now appear to be saying, "MUST include forever".